### PR TITLE
Series.is_unique method implementation

### LINKED ISSUE #1265: is_unique function implementation

### PR DESCRIPTION
**Pull request by amyskov**
Signed-off-by: Alexander Myskov <alexander.myskov@intel.com>

## What do these changes do?

- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #1265  
- [x] tests added and passing

### CODE REVIEW

File: modin/pandas/series.py [hunk 1]
```diff
@@ -1329,11 +1329,7 @@ def is_monotonic_increasing(df):
 
     @property
     def is_unique(self):
```
Comment (anmyachev) [inline modin/pandas/series.py:1331]: @amyskov plz add docstring compatible with numpydoc standard (#1411)

File: modin/pandas/test/test_series.py [hunk 2]
```diff
@@ -3617,3 +3617,32 @@ def test_str_isdecimal(data):
     else:
         modin_result = modin_series.str.isdecimal()
         df_equals(modin_result, pandas_result)
+
+
+@pytest.mark.parametrize("data", test_string_data_values, ids=test_string_data_keys)
```
Comment (devin-petersohn) [inline modin/pandas/test/test_series.py:3622]: Why were these tests added for `nunique`?

File: modin/pandas/series.py (hunk 1 above)
Comment (amyskov) [reply modin/pandas/series.py:1331]: @anmyachev, numpy docstring is added.

File: modin/pandas/test/test_series.py (hunk 2 above)
Comment (amyskov) [reply modin/pandas/test/test_series.py:3622]: After is_unique implementation, two functions became uncovered by testing, so these new tests fix this problem.

Comment (devin-petersohn) [reply modin/pandas/test/test_series.py:3622]: 👍

**Pull request merged**